### PR TITLE
add support for ech_config_path

### PR DESCRIPTION
**Pull request by saldry**

### CODE REVIEW

File: htdocs/luci-static/resources/view/homeproxy/node.js [hunk 1]
```diff
@@ -1050,8 +1050,20 @@ function renderNodeSettings(section, data, features, main_node, routing_mode) {
 		o.default = o.disabled;
 		o.modalonly = true;
 
-		o = s.option(form.DynamicList, 'tls_ech_config', _('ECH config'));
+		o = s.option(form.Value, 'tls_ech_config_path', _('ECH config path'),
+			_('The path to the ECH config, in PEM format. If empty, load from DNS will be attempted.'));
+		o.value('/etc/homeproxy/certs/client_ech_conf.pem');
 		o.depends('tls_ech', '1');
+		o.validate = L.bind(hp.validateCertificatePath, this);
+		o.rmempty = false;
```
Comment (1715173329) [inline htdocs/luci-static/resources/view/homeproxy/node.js:1058]: drop

File: root/etc/homeproxy/scripts/generate_client.uc [hunk 2]
```diff
@@ -241,7 +241,7 @@ function generate_outbound(node) {
 				enabled: true,
 				dynamic_record_sizing_disabled: (node.tls_ech_tls_disable_drs === '1'),
 				pq_signature_schemes_enabled: (node.tls_ech_enable_pqss === '1'),
-				config: node.tls_ech_config
```
Comment (1715173329) [inline root/etc/homeproxy/scripts/generate_client.uc:244]: should keep this for backwards compatibility

**Review by 1715173329 — CHANGES_REQUESTED** [review]

File: root/etc/homeproxy/scripts/generate_client.uc (hunk 2 above)
Comment (saldry) [reply root/etc/homeproxy/scripts/generate_client.uc:244]: updated

**Review by 1715173329 — APPROVED** [review]

**Comment by 1715173329** [comment]
Merged, thank you!

**Pull request merged**